# HG changeset patch
# User jwe
# Date 751590067 0
#      Mon Oct 25 23:01:07 1993 +0000
# Node ID 2c7bce68e3d8f9fd528a34368a297ff605be7bc8
# Parent  7822345086862c6ffa0f506570cf33a778236022
[project @ 1993-10-25 23:01:07 by jwe]
(compress_suffixes): Choose gunzip for .gz files.

diff --git a/info/filesys.c b/info/filesys.c
--- a/info/filesys.c
+++ b/info/filesys.c
@@ -72,16 +72,17 @@ static char *info_suffixes[] = {
   "-info",
   (char *)NULL
 };
 
 static COMPRESSION_ALIST compress_suffixes[] = {
   { ".Z", "uncompress" },
   { ".Y", "unyabba" },
   { ".z", "gunzip" },
+  { ".gz", "gunzip" },
   { (char *)NULL, (char *)NULL }
 };
 
 /* The path on which we look for info files.  You can initialize this
    from the environment variable INFOPATH if there is one, or you can
    call info_add_path () to add paths to the beginning or end of it.
    You can call zap_infopath () to make the path go away. */
 char *infopath = (char *)NULL;
